src/lib/agent/core/Agent.ts

```
@@ -0,0 +1,367 @@
+import { AgentConfig, AgentInstructions, AgentResult } from './AgentConfig'
+import { AgentContext } from './AgentContext'
+import { ConversationHistory } from '../history/ConversationHistory'
+import { Message } from '../history/Message'
```

ESLint

[error] 'Message' is defined but never used.

(@typescript-eslint/no-unused-vars)